scripts/create_test_customer.py

```diff
@@ -45,23 +45,33 @@
 
 import argparse
 import os
+import re
 import sys
 from base64 import b64encode
 from json import dumps as json_dumps
 from uuid import uuid4
 
+from yaml import add_constructor, add_implicit_resolver, safe_load, SafeLoader
 import psycopg2
-from yaml import load
-try:
-    from yaml import CLoader as Loader
-except ImportError:
-    from yaml import Loader
 import requests
 
 BASEDIR = os.path.dirname(os.path.realpath(__file__))
 DEFAULT_CONFIG = BASEDIR + '/test_customer.yaml'
 SUPPORTED_PROVIDERS = ['aws', 'ocp', 'azure']
 
+ENV_VARIABLE_MATCHER = re.compile(r'\$\{([^}^{]+)\}')
+
+def variable_constructor(loader, node):
+    """Extract the matched value, return the environment variable or the default."""
+    value = node.value
+    match = ENV_VARIABLE_MATCHER.match(value)
+    env_var = match.group()[2:-1].split(':')
+    if os.getenv(*env_var):
+        return os.getenv(*env_var)
+    raise ValueError("Environment variable is not defined and no default value given.")
+
+add_implicit_resolver('!ENV', ENV_VARIABLE_MATCHER, None, SafeLoader)
+add_constructor('!ENV', variable_constructor, SafeLoader)
```

Comment:
Let's leverage `argparse` for this, as the script already uses it. In a perfect world, the script would expose most of the options in the YAML config as CLI args. You don't have to go quite that far with it for this PR. I'm okay if you just add the couple of options most important to you into the `argparse` setup and leave the rest for a future PR.

```suggestion
parser.add_argument('--whatever-param', default=os.environ.get('WHATEVER_PARAM', 'Some Default Value'))
```